examples/specs/label_multi_line.vl.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
line_color_label

Reply:
We already have a `line_color_label` example.
Its this one: https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JAKyUAVhDYA7EABoQAEzjQATjRyZ289AFkGgnfwhxNagATDZcS7CTrMlgO50Y1pEziCIlRSpImCiooAzqgmiBwZyYbFAA1n5QEMQgAL5KmOpIshAAZmzqCGgA2qD5NMbmUcpBDAiUEACeCExsggCEALzdAOQAkgBC+n0ZALpKcLJQbMo0smRooLOCRcsglT7KUS1tHQGYzThwUbJsCAtIkUqC5NM7qLJGgumZIIJIzTWo5SDI6gSURspymMzmCyWoRAAA8NltBI9oqCQEcTlFMIgcEVroc6HdakFTu9mvCaNsojhNFAUWjTugAI4MXI6YI6Uh4zAE9BUmg0jJvBSgaazeaLDZw6FIMhkdR8TFRZBwpQIpF1BVZY701FYnGREkbaWy+X00COMhKwkK96qynU2laqJMll0II0DmCj5fH5-AFA6F0qJQGjqKAEwWgP0bQPoTEwzABa40Mh6D5wfIJpTKSUAFneIoh4oDcHjZIp6D27RuqMd6HOl1kuLe6XGLaUs1klShoGI5JcGwg2TYCXpz0ErzeQA
Also, our label-encoding version cannot replace this version because we cannot encode color into the text label.

Reply:
> Also, our label-encoding version cannot replace this version because we cannot encode color into the text label.

Why shouldn't we support this? 

It will be so confusing why we have so many similar label examples, but done in different ways due to limitations of the `label` encodings.